integration_tests/test_after.py

```diff
@@ -77,8 +78,9 @@ def test_build_with_missing_dependencies(self):
         exception = self.assertRaises(
             subprocess.CalledProcessError,
             self.run_snapcraft, 'build')
-
-        self.assertThat(exception.returncode, Equals(1))
```

Comment:
i.e. bring this back, but make it 2.

Reply:
Done (confusingly not visible on the Conversation page).